main.go

```diff
@@ -109,37 +96,30 @@ func generateCrdDocs(configFilePath string) error {
 			return microerror.Mask(err)
 		}
 
-		// Collect our own CRD YAML files
-		thisCRDFolder := clonePath + "/" + crdFolder
-		err = filepath.Walk(thisCRDFolder, func(path string, info os.FileInfo, err error) error {
-			if strings.HasSuffix(path, ".yaml") {
-				crdFiles[path] = true
-			}
-			return nil
-		})
-		if err != nil {
-			return microerror.Mask(err)
-		}
-
-		// Collect upstream CRD YAML files
```

Comment:
The differentiation between own and upstream CRDs wasn't necessary and could be removed.